go/README.md

```diff
@@ -26,7 +26,7 @@ instructions](https://keybase.io/download).
 ### Building
 
 Here's how to build the command line client on Linux or OSX. You need to
-have both Git and **Go 1.15 or higher** installed. (Run `go version` to
+have both Git and **Go 1.17 or higher** installed. (Run `go version` to
```

Comment:
I don't think 1.17 is a strict requirement here, I marked this as 1.15 because of gomobile. Although we only test against the latest version in CI I think others can still build with an earlier version if they want

Reply:
Ah I see, should I put up a PR to revert this?